### ISSUE TITLE
Ability to log every chat log

### ISSUE THREAD

**Issue by JacoboMoral** [hex-agon/chat-logger#1]
I know its a little bit weird since this plugin is a clan chat logger, but could it be possible to log every log? This is public chat and private chat as well.

**Comment by hex-agon** [comment]
It is definitely possible, do you have any suggestion for the line format?

I was thinking of maybe doing something like:
```
HH:MM:ss {PUBLIC} PlayerName: message
HH:MM:ss {PRIVATE} PlayerName: message
HH:MM:ss {CLAN} [ChatName] PlayerName: message
```

**Comment by JacoboMoral** [comment]
I was actually thinking to split them into separate files, so each one would only contain one type of chat log. This would remove the necessity to indicate the type between brackets, since it would appear in the file name.

Anyway that format is just fine, yea. Maybe add rank of the speaker on clan chatvmessage. And the person to whom the private message is sent to.

**Comment by hex-agon** [comment]
Resolved in 1877bfe7c65e065423737e9a0acd2c1340ae0757
